### PR TITLE
Arrow functions slides

### PR DESCRIPTION
**Pull request by nachosoloaga**

### CODE REVIEW

File: src/slides/class3/slides/ArrowFunctions.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        Una{" "}
+        <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions">
+          arrow function
+        </a>{" "}
+        es una alternativa un poco más compacta a una{" "}
+        <strong>function expression</strong> tradicional, aunque tiene algunas
+        limitaciones y no puede utilizarse en todos los casos. Estas
+        limitaciones son principalmente las siguientes:{" "}
+        <UnorderedList fontSize={30}>
+          <ListItem>
```
Comment (JulianPasquale) [inline src/slides/class3/slides/ArrowFunctions.jsx:28]: I'm not sure about the order of the slides. We should add classes before this, so people will have more context about what `super` and `this` means in the JS world

File: src/slides/class3/slides/ArrowFunctions.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            No tienen sus propios enlaces a this o super y no se deben usar como
+            métodos dentro de un objeto o clase.
+          </ListItem>
+          <ListItem>
+            No son aptas para los métodos call, apply y bind, que se basan en
```
Comment (JulianPasquale) [inline src/slides/class3/slides/ArrowFunctions.jsx:33]: Can we add some links about how `call`, `apply` and `bind` work? Maybe it worth their own slide, dunno

**Review by JulianPasquale — APPROVED** [review]
Nice! 🚀 

QQ: Why this PR points to the `module` branch? With this new folders structure, we should not create dependencies between branches, so we can merge any PR even if others are blocked - and we can just switch the order in `class3/index.jsx` if needed when rebasing

**Comment by nachosoloaga** [comment]
@JulianPasquale yes that's right! The thing is I created each branch based on the previous one, so I made the base for this PR the previous branch in order to not see all modifications made by previous branches 😅

I'll correct that on next PRs 😄

File: src/slides/class3/slides/ArrowFunctions.jsx (hunk 1 above)
Comment (nachosoloaga) [reply src/slides/class3/slides/ArrowFunctions.jsx:28]: That sounds good, next PR is classes indeed, so maybe we can change the order then.

File: src/slides/class3/slides/ArrowFunctions.jsx (hunk 2 above)
Comment (nachosoloaga) [reply src/slides/class3/slides/ArrowFunctions.jsx:33]: Yep of course! What do you think about finishing first with all topics written down on the google doc, and then see if we can add a couple of slides regarding this methods?

Comment (JulianPasquale) [reply src/slides/class3/slides/ArrowFunctions.jsx]: Sure, sounds great. Lets add at least a comment in the doc so we don't forget to add it

**Pull request merged**